ucsmsdk/ucssession.py

```diff
@@ -463,13 +510,7 @@ def _login(self, auto_refresh=False, force=False):
         Returns:
             True on successful connect
         """
-
-        from .mometa.top.TopSystem import TopSystem
-        from .mometa.firmware.FirmwareRunning import FirmwareRunning, \
-            FirmwareRunningConsts
-        from .ucscoremeta import UcsVersion
         from .ucsmethodfactory import aaa_login
-        from .ucsmethodfactory import config_resolve_dn
 
         self.__force = force
 
```

Comment:
@ragupta-git If it was not already present, I would not have changed that.. I have just refactored code for readability and usability. So, may be we should track it separately.